### PR TITLE
fixes #284 - prepared CountryTweetMap app for store listing

### LINKED ISSUE #284: Prepare CountryTweetMap app for store listing.
Prepare CountryTweetMap app so that it can be presented on store listing page. Also expose the app on Loklak app wall.

### PR DESCRIPTION
**Pull request by djmgit**
Fixes #284, prepared CountryTweetMap app for store listing and
exposed it on loklak app wall.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link: http://loklakapps.surge.sh/details.html?q=CountryTweetMap

@kavithaenair @SKrPl @Achint08 please review

File: CountryTweetMap/getStarted.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+CountryTweetMap is an application which allows you to visualise
+tweet frequency by plotting it on a map. Number of tweets are
+categorised into high, medium and low. A Country v/s number of
+tweets statistics is also provided.
+
+### How to use this app
+
+Explore the live app by clicking the "Try now" button. Enter an query
```
Comment (Achint08) [inline CountryTweetMap/getStarted.md]: a query

**Comment by djmgit** [comment]
@Achint08 made changes, please re-review.

File: CountryTweetMap/getStarted.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+tweet frequency by plotting it on a map. Number of tweets are
+categorised into high, medium and low. A Country v/s number of
+tweets statistics is also provided.
+
+### How to use this app
+
+Explore the live app by clicking the "Try now" button. Enter a query
+word in the input field provided. Optionally, enter start and end dates
+for filtering. Enter number of tweets you want to plot. Press the search
+button. The location of the tweets containing the query word enterd will
```
Comment (kavithaenair) [inline CountryTweetMap/getStarted.md]: typo here

**Comment by djmgit** [comment]
@kavithaenair made the changes, please re-review.

**Comment by kavithaenair** [comment]
@djmgit rebase the branch

**Pull request merged**